src/cryptoadvance/specter/util/requests_tools.py

```diff
@@ -4,6 +4,7 @@
 from requests.exceptions import ConnectionError, HTTPError
 from urllib3.exceptions import NewConnectionError
 from json.decoder import JSONDecodeError
+from simplejson.errors import JSONDecodeError as SjJSONDecodeError
```

Comment:
Why is simplejson nedded? Is the json module insufficient?

Reply:
Good point. I've mixed it up with spectrum where simplejson is used for some reason. If you have dependencies of both projects in one virtualenv and simplejson is installed, it will automatically be used by the requests library.
I've removed it here!